test/test_other.py

```diff
@@ -14018,3 +14018,18 @@ def test_hello_world_argv(self):
 
   def test_arguments_global(self):
     self.emcc(test_file('hello_world_argv.c'), ['-sENVIRONMENT=web', '-sSTRICT', '--closure=1', '-O2'])
+
+  @parameterized({
+    'no_std_exp':   (['-DEMMALLOC_NO_STD_EXPORTS'],),
+    # When we let emmalloc build with the standard exports like malloc,
+    # emmalloc == malloc.
+    'with_std_exp': (['-DTEST_EMMALLOC_IS_MALLOC'],),
+  })
+  def test_emmalloc_in_addition(self, args):
```

Comment:
I don't love the name of this test (or the `NO_STD_EXPORTS`) but I'm struggling to come up with anything better.

I'm not sure you need the `with_std_exp` variant of this test since that is the default and tested elsewhere.    Can we simplify this test by just verifying that it works as expected in the "NO_EXPORT" case?

Reply:
Maybe `test_emmalloc_explicit` or `test_emmalloc_standalone`?   I don't really love them either.

Reply:
I considered "standalone", but I felt it is too similar to "standalone wasm" and so it's confusing. I could rename to "explicit" but I thing "in addition" is slightly more clear?

I agree `with_std_exp` is tested elsewhere, but I like that testing both here in relation to that one flag shows the flag exactly controls that behavior. So if we ever get it wrong it won't be a random test that fails, but here. I'd also worry about us removing/modifying the other tests, though that seems like likely.

Reply:
How about "alongside" instead of "in addition" or "explicit"? This does test that we can have emmalloc alongside another malloc at the same time.